src/source/Signaling/LwsApiCalls.c

```diff
@@ -2012,9 +2012,13 @@ STATUS receiveLwsMessage(PSignalingClient pSignalingClient, PCHAR pMessage, UINT
         DLOGW("Failed to validate the ICE server configuration received with an Offer");
     }
 
+#ifdef KVS_USE_SIGNALING_CHANNEL_THREADPOOL
```

Comment:
Wondering if we really want to set this up in an ifdef block. What do you think about using threadpool always and set the MIN_MAX to 3,5 by default and allow we want this to be customer configurable? Because webrtc would create a min of 3 threads anyways - one for offer, one for answer and one per ice-candidate

Reply:
I've turned it on by default, but went with the ifdef block to preserve the previous implementation. I think a threadpool is better for most use cases, but if a user finds their system has the resources to handle bringing up and tearing down threads quickly then it threadpool is just needless overhead.